snap/info_snap_yaml.go

```diff
@@ -375,6 +376,7 @@ func setAppsFromSnapYaml(y snapYaml, snap *Info, strk *scopedTracker) error {
 			Before:          yApp.Before,
 			After:           yApp.After,
 			Autostart:       yApp.Autostart,
+			DoNotTrack:      yApp.DoNotTrack,
```

Comment:
I'd like to suggest that we pick a different name; this collides with a (much maligned) HTTP privacy header https://en.wikipedia.org/wiki/Do_Not_Track

We should be more clear that this variable controls several behaviours:

- running applications prevent automatic upgrades
- prevent automatic upgrades from disrupting running programs

I'm sorry to say I don't have a great suggestion for the name, but 'do-not-track' doesn't say either of these two meanings to me.

Reply:
I'm open to suggestions. Since this is called app tracking internally I used the "do not track" name that I was already familiar with, but I agree some people may find it confusing.

Reply:
To avoid the confusion with the HTTP header, renaming to `Untracked` would be enough. I'm not sure I understand the difference between the two behaviours mentioned by @setharnold; if they are different enough, we might even want to use two different flags. Otherwise, maybe `RunningBlocksUpgrade` or `BlockUpgradeWhileRun`, though maybe they are too long.

Reply:
I think `unracked` is enough, it does what is says on the tin: disable tracking.

Reply:
@mardy I've force-pushed as "untracked", have a look.